src/org/zalando/stups/essentials/api.clj

```diff
@@ -61,7 +84,9 @@
 (defn read-resource-types
   "Provides a list of all resource types"
   [_ request db]
-  (u/require-internal-user request)
+  (if (:tokeninfo request)
+    (u/require-realms #{"services" "employees"} request)
+    (log/warn "Could not validate authorization due to missing tokeninfo. Set HTTP_TOKENINFO_URL to enable full validation"))
```

Comment:
I dunno, this is what was there before so I did it everywhere for consistency. I don't really care, if you like I change this to `(when ...)`.